FashionConnect_ui/src/components/PasswordChange/PasswordChange.jsx

```diff
@@ -0,0 +1,72 @@
+import "./PasswordChange.css"
+import React from "react"
+import { useState } from "react"
+import { useNavigate } from 'react-router-dom';
+
+
+export default function PasswordChange({ userID }) {
+    const [password, setPassword] = useState("")
+    const [secondPassword, setSecondPassword] = useState("")
+    const navigate = useNavigate();
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+
+        if (!password === secondPassword) {
+            alert("Password do not match");
+            return
+        }
+        try {
+            // Make the  API request
+            const response = await fetch(`http://localhost:3000/password-change`, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify({ userID, password }),
+                credentials: 'include'
+            });
+
+            if (response.ok) {
+                // Navigate to the home page after successful password change
+                navigate('/');
+            } else {
+                // Handle the password change failure case
+                alert('Login failed');
+            }
+        } catch (error) {
+            // Handle any network or API request errors
+            alert('Login failed: ' + error);
+        }
+    };
+
+    return (
+        <div className="signup-form-container">
+            <form className="signup-form" onSubmit={handleSubmit}>
+                <h2>Sign Up</h2>
```

Comment:
This shouldn't be Sign up right? I think something more like Change Password.

Reply:
yeah!
I mistakenly overlooked this.